src/Google.Api.Gax/RetrySettings.cs

```diff
@@ -61,8 +61,8 @@ public sealed class RetrySettings
             {
                 RetryBackoff = RetryBackoff?.Clone(),
                 TimeoutBackoff = TimeoutBackoff?.Clone(),
+                RetryFilter = RetryFilter,
```

Comment:
Any reason for this change? Only one in the file, so probably worth reverting for a simpler commit.